From: Abhijeet Joglekar <abjoglek@cisco.com>
Subject: libfc: Remove debug print statement, too verbose
References: bnc #459142

fc_disc_lookup_rport() gets called for incoming Plogis. Since libFC currently
does not handle incoming Plogis from initiators, the rport is not found in the
discovery list. This message gets printed a lot if fc_disc_debug == 1, and
hides other discovery messages. It is probably not needed.

Signed-off-by: Abhijeet Joglekar <abjoglek@cisco.com>
Acked-by: Bernhard Walle <bwalle@suse.de>
---

 drivers/scsi/libfc/fc_disc.c |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)


diff --git a/drivers/scsi/libfc/fc_disc.c b/drivers/scsi/libfc/fc_disc.c
index 190e6bb..f30ce90 100644
--- a/drivers/scsi/libfc/fc_disc.c
+++ b/drivers/scsi/libfc/fc_disc.c
@@ -101,12 +101,8 @@ struct fc_rport *fc_disc_lookup_rport(const struct fc_lport *lport,
 		}
 	}
 
-	if (!disc_found) {
-		FC_DEBUG_DISC("The rport (%6x) for lport (%6x) "
-			      "is not maintained by the discovery layer\n",
-			      port_id, fc_host_port_id(lport->host));
+	if (!disc_found)
 		found = NULL;
-	}
 
 	return found;
 }
